api/schema.go

```diff
@@ -37,10 +35,50 @@ func NewSchemaService(ar *Runner) *SchemaService {
 		log.Panicf("Can't parse schema from '%s' : %s", path, err)
 	}
 
+	ss.Data = Yaml2Json(ss.Data)
+
 	return ss
 }
 
-func (s *SchemaService) Get(r *http.Request, args *SeedArgs, _ *rpc.RequestBody, reply *MAP) error {
+func Yaml2Json(in interface{}) interface{} {
+	switch i := in.(type) {
+	case map[interface{}]interface{}:
+		ret := map[string]interface{}{}
+		for k, v := range i {
+			switch ii := k.(type) {
+			case string:
+				ret[ii] = Yaml2Json(v)
+			default:
+				log.Panicf("TYPE1 %#v", k)
```

Comment:
It would be nice to have more info in log message, like what type we expected to get.

Reply:
ok